.github/workflows/main_branches_ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI Check for Mobile-Wallet
```

Comment:
Same as the last file, rename this. Also IMO we should rename this file to something like master_dev_ci.

Reply:
Okay, what do you think about the name: `Mobile Wallet CI[Master/Dev]`

Reply:
Yes sure.